src/engine/renderer/tr_vbo.cpp

```diff
@@ -554,24 +554,19 @@ void R_BindNullIBO()
 }
 
 static void R_InitGenericVBOs() {
+	/* These values make possible to use either the legacy quad rendering
```

Comment:
To explain the values chosen here, you could say that they are chosen to specify a full-screen rectangle for screen-space shaders without a projection matrix. The values here don't influence the triangle but it's good to mention that this is a fallback when the `gl_VertexID`-based triangle can't be used.